### DISCUSSION TITLE
For "GuildMember.timeout" what is the maximum amount of time allowed?

### DISCUSSION THREAD

**Discussion by VmiteOCE** [discordjs/discord.js#7241] [Q&A] [answered]
Just curious if there's a realistic time limit for bots using the .timeout as I know users can only give a 1 week timeout.

**Comment by iCrawl** [comment, marked as answer]
28 days.

**Reply by VmiteOCE** [reply]
Is this said anywhere in the documentation that I might have over looked?

**Reply by Jam-Manbo** [reply]
not in discord.js docs but you can lookup in [developer portal's docs](https://discord.com/developers/docs/resources/guild#modify-guild-member)
you can check `communication_disabled_until` part.
